app/flatpak-builtins-config.c

```diff
@@ -63,18 +63,57 @@ looks_like_a_language (const char *s)
   return TRUE;
 }
 
+static gboolean
+looks_like_a_region (const char *s)
+{
+  g_auto(GStrv) locale = g_strsplit (s, "_", 3);
+  int i;
+  int len;
+
+  if (locale[1] != NULL)
+    {
+      len = strlen (locale[1]);
+
+      // This can be either EN or Latn
```

Comment:
Actually the EN example may be weird because I don't think we have en_EN, but this is to match, let's say en_GB - which the second section should be a 2 letter upper case, or the Latn/Cyrl for the Alphabet descriptor.

Reply:
Yeah, better to say GB then